yksmp/src/lib.rs

```diff
@@ -247,12 +247,16 @@ impl StackMapParser<'_> {
             self.read_u8();
             let size = self.read_u16();
             let dwreg = self.read_u16();
-            let extrareg = self.read_u16();
+            self.read_u16();
+            let mut extras = Vec::new();
```

Comment:
I think you can do `let extras = 0..self.read_u16().map(|_| self.read_i16()).collect::<Vec<_>>()` or similar if you'd prefer.

Reply:
Personally I find that harder to understand on a glance.

Reply:
Up to you!